File: audiophile-client/src/app/shared/models/product.ts
export interface IProduct {
  id: number;
  slug: string;
  name: string;
  image: {
    mobile: string;
    tablet: string;
    desktop: string;
  };
  category: string;
  new: boolean;
  price: number;
  description: string;
  features: string;
  includes: [
    {
      quantity: number;
      item: string;
    }
  ];
  gallery: {
    first: {
      mobile: string;
      tablet: string;
      desktop: string;
    };
    second: {
      mobile: string;
      tablet: string;
      desktop: string;
    };
    third: {
      mobile: string;
      tablet: string;
      desktop: string;
    };
  };
  others: [
    {
      slug: string;
      name: string;
      image: {
        mobile: string;
        tablet: string;
        desktop: string;
      };
    }
  ];
  display: {
    mobile: string;
    tablet: string;
    desktop: string;
  };
}

File: Audiophile.BlazorUI/Models/Product.cs
namespace Audiophile.BlazorUI.Models
{
    public class Product
    {
        public int Id { get; set; }
        public string Slug { get; set; }
        public string Name { get; set; }
        public Image Image { get; set; }
        public string Category { get; set; }
        public bool New { get; set; }
        public decimal Price { get; set; }
        public string Description { get; set; }
        public string Features { get; set; }
        public List<Include> Includes { get; set; }
        public Gallery Gallery { get; set; }
        public List<Other> Others { get; set; }
    }
}
